Expensify/react-fast-pdf
# PDF preview component

### Details



This is a PR where we introduce a MVP version of `react-fast-pdf` package. The package codebase itself has a basic configuration to read and preview a _PDF_ file. The `example` has a basic template how to use the package. 

### Related Issues



https://github.com/Expensify/App/issues/22998

### Manual Tests



1. Clone the repo.
2. Install deps. for both **package** and **example**: `npm install && cd example && npm install && cd ..`.
3. Run a script - `cd example && npm run start && cd ..` - of the **example**.
4. Verify that **Webpack** launches successfully.
5. Verify that it opens a tab.
6. Verify that the PDF previewer works. 

### Linked PRs
N/A



<img width="1728" alt="Screenshot 2023-09-20 at 19 51 08" src="https://github.com/Expensify/react-fast-pdf/assets/57314631/6f08b45c-6b9d-4b46-bffb-5d5f07345e45">


https://github.com/Expensify/react-fast-pdf/assets/57314631/c73dfb1d-1194-4a80-9129-40460d981b33
